docs/reference/query-dsl/nested-query.asciidoc

```diff
@@ -114,4 +116,125 @@ If `false`, {es} returns an error if the `path` is an unmapped field.
 
 You can use this parameter to query multiple indices that may not contain the
 field `path`.
---
\ No newline at end of file
+--
+
+[[nested-query-notes]]
+==== Notes
+
```

Comment:
Why call this "Notes"? Do you even need this level heading? In general, we should avoid "dead" heads with no content.

Reply:
Some day, we should fix that. :-)